src/cli.rs

```diff
@@ -150,6 +152,28 @@ pub struct Cli {
         default_value = "#",
     )]
     pub topic: String,
+
+    /// Path to the TLS client certificate file.
+    /// Used together with --client-key to enable TLS client authentication.
+    /// The file has to be a DER-encoded X.509 certificate serialized to PEM.
+    #[arg(
+        long = "client-cert",
```

Comment:
`long` assumes the name from the variable, which is less error prone afterwards to forget to rename something afterwards. Sorry for not writing that after my last change request here.

```suggestion
        long,
```